### ISSUE TITLE
Special color for TODO comments

### ISSUE THREAD

**Issue by mgarin** [bobbylight/RSyntaxTextArea#18]
TODOs highlighting might be a good addition.

Here is an example of TODOs in Java code (in RSyntaxTextArea):
![scr1](https://f.cloud.github.com/assets/2361389/1164703/88c27766-205e-11e3-828e-10f1b20632de.png)

And here is how IntelliJ IDEA highlights it:
![src2](https://f.cloud.github.com/assets/2361389/1164708/a0876b86-205e-11e3-9809-f56e4d79b082.png)

Basically - it highlights anything that goes after "todo" (ignoring case) mark on its line inside any comment with blue color.

Color can be moved to RSyntaxTextArea style settings i guess.

**Comment by JonSn0w** [comment]
I totally agree. After becoming so accustomed to IntelliJ, I've noticed that  "TODO" & "FIXME"  highlighting is a small thing that can be hard to go without.
